apps/examples/src/examples.tsx

```diff
@@ -7,22 +7,99 @@ export type Example = {
 	path: string
 	codeUrl: string
 	hide: boolean
+	category: 'basic' | 'editor' | 'ui' | 'collaboration' | 'data/assets' | 'shapes/tools' | null
```

Comment:
i don't think null should be a category option? is there a good reason for that?